Source/santametricservice/Formats/SNTMetricMonarchJSONFormat.m

```diff
@@ -160,7 +161,9 @@ - (void)encodeValueAndStreamKindFor:(NSString *)metricName
  * formatMetric translates the SNTMetricSet metric entries into those consumable by Monarch.
  **/
 
-- (NSDictionary *)formatMetric:(NSString *)name withMetric:(NSDictionary *)metric {
+- (NSDictionary *)formatMetric:(NSString *)name
```

Comment:
The name of this method is confusing because of the differing uses of metric, perhaps `formatMetric:withValue:andEndtimestamp:` or `formatMetricNamed:withMetric:endTimestamp:`?

Reply:
It wasn't the end timestamp argument I was concerned with, this still has `formatMetric:withMetric:`, which doesn't read well. I recognize that wasn't added in this change, but changing the signature highlighted it.